fastlane/Fastfile

```diff
@@ -270,19 +268,20 @@ platform :ios do
 
   desc 'Increment build number based on version in App Store Connect'
   lane :increment_build_number_for_version do |options|
-    if options[:app_identifier]
-      app_identifier = options[:app_identifier]
```

Comment:
This was also causing an issue, since `app_identifier` wasn't being used. I've updated this to pass it through to the `latest_testflight_build_number` function, and confirmed that this still works as expected when `app_identifier` isn't passed as an option.